OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,43 @@
+Example of Apache Web Server with WebLogic 12.2.1.3.0 Plugin
+=====
+This sample shows how to run a load balancer for a WebLogic cluster, from inside a container in the same network as the WLS Cluster.
+
+# How to build image
+Download file **Oracle WebLogic Server Proxy Plugins 12.2.1** [fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip](http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html) (see .download file)
+
+You may build this image with:
+
+        $ docker build -t 12213-apache .
+
+or
+
+        $ sh build.sh
+
+Oracle developers can use build_internal.sh, which set up HTTP/HTTPS proxies.
+
+        $ sh build_internal.sh
+
+# How to run container
+Run this image by calling:
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+The values of **WEBLOGIC_CLUSTER** must be valid, existing containers running WebLogic servers.
+
+## Example
+Start an AdminServer from the **12213-domain** sample by calling:
+
+        $ docker run -d --name wlsadmin -h wlsadmin 12213-domain
```

Comment:
Start a container from the WebLogic install image. You can override the default values of the following parameters during runtime with the -e option:
        ADMIN_NAME (default: AdminServer)
        ADMIN_PORT (default: 7001)
        ADMIN_USERNAME (default: weblogic)
        ADMIN_PASSWORD (default: Auto Generated)
        DOMAIN_NAME (default: base_domain)
        DOMAIN_HOME (default: /u01/oracle/user_projects/domains/base_domain)

NOTE To set the DOMAIN_NAME, you must set both DOMAIN_NAME and DOMAIN_HOME.

    $ docker run -d -e ADMIN_USERNAME=weblogic -e ADMIN_PASSWORD=welcome1 -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain -e DOMAIN_NAME=abc_domain store/oracle/weblogic:12.2.1.3